python/semantic_kernel/connectors/ai/hugging_face/__init__.py

```diff
@@ -1,6 +1,6 @@
 # Copyright (c) Microsoft. All rights reserved.
 
-from semantic_kernel.connectors.ai.hugging_face.hf_request_settings import (
+from connectors.ai.hugging_face.request_settings.hf_request_settings import (
```

Comment:
All the other imports are using the absolute name with `semantic_kernel.`, shouldn't this follow the same?

Reply:
yeah, that is a mistake